docs/protocol.md

```diff
@@ -662,7 +673,233 @@ Response (in case of success)
 ```
 
 </details>
+
+## Sending a heartbeat message to servers
+
+🧭 **RPC Message** > **Query** > **Heartbeat**
+
+A server sends a heartbeat to its peers in order to attest that: 
+
+- it is alive
+- it received a certain set of messages
+
+Heartbeat are to be sent based on channel activity and time intervals.  
```

Comment:
i.e. after a given threshold of messages have been sent or a given amount of time has passed since the last heartbeat? Or do I misunderstand this sentence? :)

Assuming this is what's meant, when is the first heartbeat sent / when does this timer start?

Reply:
The idea is to have a combination of both. Let's say the threshold of messages is reached before the timeout is expired, then we can send the heartbeat and then restart the timer. This is just a possible approach, we discussed some others today, but for now I guess we will start by simply implementing a very simple approach (every X ms send the heartbeat). The timer can start when we receive the first createLao from the organizer or an heartbeat from another server, but also this one is a wip approach. In that sense we will add details to the doc as we plan a clear and more defined working strategy.